### ISSUE TITLE
[all beats] default paths seem to be ignored at the moment

### ISSUE THREAD

**Issue by 111andre111** [elastic/beats#12571]
I have tested under RHEL7 and Ubuntu 16 so far quite all beats I know and it seems like default directory layout is ignored at the moment, only with the correct parameters for instance loading dashboards works..

For confirmed bugs, please report:
- Version: 7.1.17
- Operating System: RHEL7, Ubuntu 16
- Discuss Forum URL:
    https://www.elastic.co/guide/en/beats/auditbeat/current/directory-layout.html#_default_paths
    https://www.elastic.co/guide/en/beats/filebeat/current/directory-layout.html#_default_paths
    https://www.elastic.co/guide/en/beats/heartbeat/current/directory-layout.html#_default_paths
    https://www.elastic.co/guide/en/beats/metricbeat/current/directory-layout.html#_default_paths
    https://www.elastic.co/guide/en/beats/packetbeat/current/directory-layout.html#_default_paths
    https://www.elastic.co/guide/en/beats/journalbeat/current/directory-layout.html#_default_paths
    
- Steps to Reproduce:
Regarding the documentation, there is for quite all beats a default directory setup, which should be regarding documentation the default values as well.
But seems, like this is not working at the moment.

Please have a look what I did. To be easier over all beats, I have parametrized everything with variables:
```
### first choose one of the beats here
### prerequisite: elasticsearch and kibana are installed, I just used a fresh vagrant 
### for errors, I just replaced with the variable as well
environment
    export beatname=file
    export beatname=metric
    export beatname=packet
    export beatname=audit
    experimental: export beatname=journal
### and now install and test loading dashboard
    sudo yum install ${beatname}beat
    sudo /usr/share/${beatname}beat/bin/${beatname}beat setup --dashboards -v -d *
      ERROR: Exiting: error loading config file: stat ${beatname}beat.yml: no such file or directory
    cd /etc/${beatname}beat
    sudo /usr/share/${beatname}beat/bin/${beatname}beat setup --dashboards -v -d *
      ERROR: Skipping loading dashboards, No directory /usr/share/${beatname}beat/bin/kibana/7
    sudo /usr/share/${beatname}beat/bin/${beatname}beat -path.home /usr/share/${beatname}beat -path.config /etc/${beatname}beat -path.data /var/lib/${beatname}beat -path.logs /var/log/${beatname}beat setup --dashboards -v -d *
        Success

### now same with hearbeat, it has no dashboard but only template or ilm-policy, but without changing directory, doesn't work as well
    export beatname=heart
    sudo yum install ${beatname}beat-elastic
    sudo /usr/share/${beatname}beat/bin/${beatname}beat setup --template -v -d *
      ERROR: Exiting: error loading config file: stat heartbeat.yml: no such file or directory
    cd /etc/${beatname}beat
    sudo /usr/share/${beatname}beat/bin/${beatname}beat setup --template -v -d *
        Success

# I still tested for winlogbeat and under some circumstances, I met the situation, that by hand it loaded, but not starting the service, but this is maybe a seperate ticket
```

I have started the services using `systemctl start <servicename>` and they started smoothely, so it's always an issue of starting the beats standalone.

**Comment by andrewkroh** [comment]
The commands the user is running are not the same as what's given in the [documentation](https://www.elastic.co/guide/en/beats/auditbeat/current/load-kibana-dashboards.html).

<img width="315" alt="Screen Shot 2019-06-17 at 9 13 19 AM" src="https://user-images.githubusercontent.com/4565752/59607005-76b29580-90e0-11e9-8047-98abcb1c6f66.png">

There's a subtle difference. They are running the beat binary directly rather than running the beat wrapper script, which placed on the `PATH` at` /usr/bin/${beatname}`, that sets the `--path.{home,logs,data,config}` flags to the appropriate values.

**Comment by 111andre111** [comment]
Oh yes, this is a good point. But I cannot find about that there exists the wrapper script.

Searching for it I have found that now:
```
$ rpm -ql ${beatname}beat | grep ${beatname}beat | grep -v -E "json|yml"
/etc/init.d/metricbeat
/lib/systemd/system/metricbeat.service
/usr/bin/metricbeat
/usr/share/metricbeat/README.md
/usr/share/metricbeat/bin/metricbeat
/usr/share/metricbeat/bin/metricbeat-god
```
So at least to be seen are here in the package the init scripts and the wrapper script itself as well, interesting.

```
$ which ${beatname}beat
/usr/bin/metricbeat
$. ${beatname}beat setup --dashboards
   Success
```
And leaving the path completely out it works as well.

Good, so at least in the documentation page there is still the doc nice to have:  https://www.elastic.co/guide/en/beats/auditbeat/current/load-kibana-dashboards.html

There is the mentioning correctly about leaving out the path like for. deb and rpm. But I think it would be worth mentioning, that the `/usr/bin/${beatname}beat` is a wrapper script whereas `/usr/share/${beatname}beat/bin/${beatname}beat` is a binary. Otherwise, this can lead to misunderstanding.

**Comment by andrewkroh** [comment]
I agree that it would be good to mention the presence of the wrapper script in the RPM and Deb distributions. Perhaps we should mention this in both the Getting Started Guide and in the [default directory layout page](https://www.elastic.co/guide/en/beats/filebeat/current/directory-layout.html).

Can you please close this issue and open a new issue tagged with `docs` strictly focused on updating the documentation with information about the wrapper script?

**Comment by 111andre111** [comment]
Ok, great I will do that. Thank you for that.
